ui/react-ui/src/components/routes/home/Home.js

```diff
@@ -17,6 +18,7 @@ let Home = () => {
       <h2>Test Result Endpoints:</h2>
```

Comment:
left the same comment for @ahmadafrasiyab, that would look much better if we had some margin-top for this element =>
```<h2 style={{marginTop: '6rem'}}>Test Result Endpoints:</h2>```

Reply:
I believe youre right! thanks for the recommendation I'll make this change.